### PR TITLE
Add is crlp is enabled to callable api

### PR DESCRIPTION
**Pull request by patrick-yan-sf**
Add a new callableAPI method - isCrlpEnabled for DSO-Bridge work. 

# Critical Changes

# Changes

 - We introduced code related to integration between NPSP and the forthcoming Data Storage Optimizer product.

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by patrick-yan-sf** [comment]
**lurch: attach W-032958

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E0000087uZSQAY" target="_blank">W-032958</a>

File: src/classes/Callable_API.cls [hunk 1]
```diff
@@ -82,6 +82,10 @@ global with sharing class Callable_API implements System.Callable {
 
                 return JSON.serialize(rollupDefs,false);
 
+            } when 'crlp.iscrlpenabled' {
```
Comment (RadSFDeveloper) [inline src/classes/Callable_API.cls]: @force2b , @seanpat09 : any objection to the Callable method name?

Comment (force2b) [reply src/classes/Callable_API.cls]: Good for me.

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR.  Tests have passed.

For further testing details, see: https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E0000087uZSQAY/view

Merging PR and deleting branch.

**Pull request merged**